wagtail/admin/templates/wagtailadmin/account/change_avatar.html

```diff
@@ -0,0 +1,28 @@
+{% extends "wagtailadmin/base.html" %}
+{% load wagtailadmin_tags i18n %}
+
+{% block titletag %}{% trans "Change profile picture" %}{% endblock %}
+{% block content %}
+    {% trans "Change profile picture" as change_str %}
+    {% include "wagtailadmin/shared/header.html" with title=change_str %}
+
+    <div class="nice-padding">
+        <p>{% trans "Your current profile picture:" %}</p>
+        <p>
+            <span class="avatar square"><img src="{% avatar_url request.user size=50 %}"></span>
+        </p>
+
+        <form action="{% url 'wagtailadmin_account_change_avatar' %}" method="POST" enctype="multipart/form-data">
+            {% csrf_token %}
+
+            <ul class="fields">
+                {% for field in form %}
+                    {% include "wagtailadmin/shared/field_as_li.html" with field=field %}
+                {% endfor %}
+                <li class="submit"><input type="submit" value="{% trans 'Update' %}" class="button" /></li>
```

Comment:
Minor thing in relation in what is happening within #4455 for example. I think it would make more sense to start formatting it as: `<button class="button">{% trans 'Update' %}</button>` instead of using an `input` with a value attribute. Also the default type for a button is submit.